lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -171,7 +175,8 @@ <h3 class="course-title">
               </a>
               <div class="actions-dropdown" id="actions-dropdown-${dashboard_index}" aria-label="${_('Additional Actions Menu')}">
                 <ul class="actions-dropdown-list" id="actions-dropdown-list-${dashboard_index}" aria-label="${_('Available Actions')}" role="menu">
-                  <li class="actions-item" id="actions-item-unenroll-${dashboard_index}">
+                  % if cert_status.get('status') not in disable_unenroll_cert_states:
+                    <li class="actions-item" id="actions-item-unenroll-${dashboard_index}">
```

Comment:
This blocks the frontend. Is there a reasonable place to block on the backend as well, for a belt-and-suspenders approach?